changelog.md

```diff
@@ -1,3 +1,7 @@
+### <a name="2.11.6"></a> 2.11.6
+
+* Adds optimization to reduce header size by excluding session token for stored procedures, triggers, and user defined functions.
```

Comment:
What about mentioning the issue it fixes? I think users might not understand what the optimization is but actually look for the error they are getting and check that it's fixed.

* Fixes Header Too Long (or whatever name was the error) while doing CRUD operations on stored procedures, triggers, and user defined functions.